ni/niveristand-communications-bus-template
# Cloning - Help files build and work.

- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/niveristand-communications-bus-template/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

This fixes a couple of help bugs:
1. It removes renaming the "existing" chm and hhp files, because they don't actually exist in the source code. They were in my source due to a build I did at one point.
2. It edits the CD_CHM_FileName string constant in each Page VI to match the new clone name.

#### Edited files:

- `Clone Template Main.vi` 
-1. Added **Edit Page VIs Help CHM Constant** case to change the string constant in each VI.
-2. Removed `Rename Help Files.vi` from the **Update Help Files** case.
-3. Bonus - changed the selection mode to Folders for the Enter Destination Directory path control.
- `Edit VI Constants.vi` was renamed to `Edit VI String Constant.vi` and now has the **string label** as an input. This is to make it more generic, so it could be used to edit the Page VIs.
- `Rename Help Files.vi` was deleted, because there are no files in source to rename.
- `Consumer State.ctl` has an additional item: **Edit Page VIs Help CHM Constant**.
- `Queue Cluster.ctl` contains `Consumer State.ctl`.

### Why should this Pull Request be merged?

This will mean that Help is no longer broken in System Explorer.

### What testing has been done?

Testing on my dev VM includes:

1. Run Clone Template Main.vi to clone the source, and confirm Help works in System Explorer.
2. Run Clone Template Main.vi to clone the source to a non-default folder, and confirm Help works in System Explorer.
